p2p/transport/websocket/websocket_test.go

```diff
@@ -0,0 +1,350 @@
+package websocket
+
+import (
+	"context"
+	"crypto/rand"
+	"crypto/rsa"
+	"crypto/tls"
+	"crypto/x509"
+	"crypto/x509/pkix"
+	"fmt"
+	"io"
+	"io/ioutil"
+	"math/big"
+	"net"
+	"testing"
+	"time"
+
+	"github.com/libp2p/go-libp2p-core/crypto"
+	"github.com/libp2p/go-libp2p-core/network"
+	"github.com/libp2p/go-libp2p-core/peer"
+	"github.com/libp2p/go-libp2p-core/sec"
+	"github.com/libp2p/go-libp2p-core/sec/insecure"
+	"github.com/libp2p/go-libp2p-core/test"
+	"github.com/libp2p/go-libp2p-core/transport"
+
+	csms "github.com/libp2p/go-conn-security-multistream"
+	mplex "github.com/libp2p/go-libp2p-mplex"
```

Comment:
we should probably wait for these to be moved first.

Reply:
Why? go-libp2p depending on external repos is ok (the other way around would not).

Reply:
sure, but we'll have to remember to fix it!

Reply:
Of course. I do a `sed` for the import path every time I move a repo.

Reply:
alright, fair enough.